science/raxml/Portfile

```diff
@@ -24,6 +26,13 @@ mpi.setup
 
 use_configure       no
 
+# xcode 11 + AVX2 seems to have a bug, where stacks aren't 16-byte aligned (the C library
+# enforces different things than the compiler).
+# https://forums.developer.apple.com/thread/121887
+platform darwin 19 {
+    macosx_deployment_target 10.14
+}
+
```

Comment:
isn't this the problem what the `xcode_workaround-1.0` PG is supposed to fix, and that should be used instead (@MarcusCalhoun-Lopez)?

Reply:
There is a pull request (#7125) that should make this unnecessary.